packages/ffe-searchable-dropdown-react/less/ffe-multiselect-searchable-dropdown.less

```
@@ -0,0 +1,203 @@
+.ffe-searchable-dropdown-multiselect {
+    @check-icon-width: 24px;
+
+    &--container {
+        display: flex;
+        border-radius: var(--ffe-g-border-radius);
+        transition: all var(--ffe-transition-duration) var(--ffe-ease);
+
+        &:has(.ffe-searchable-dropdown-multiselect--input-container .ffe-searchable-dropdown-multiselect--input-field:focus) {
```

```suggestion
        &:has(.ffe-searchable-dropdown-multiselect--input-field:focus-visible) {
```

Focus visible slik att vi ikke får focus stilar på native når man klicker

---

Har oppdatert til focus-visible. Det var et forslag om å fjerne .ffe-searchable-dropdown-multiselect--input-container, men det blir ikke riktig.